WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -485,15 +493,6 @@ extension OrderDetailsViewModel {
                                                              }, onCompletion: { (result) in
             // Propagate the result to the UI
             onCompletion(result)
-
```

Comment:
This was done automatically. Now we removed it, as promised, to trigger the receipt printout when tapping the proper button in the alert